web/components/atoms/BackButton.tsx

```diff
@@ -0,0 +1,17 @@
+import { MouseEventHandler } from "react";
+
+interface Props {
+	onClick?: MouseEventHandler<HTMLButtonElement>;
+	disabled?: boolean;
+}
+
+export default function BackButton({ onClick, disabled }: Props) {
+	return (
+		<button className="group flex items-center gap-[2px] text-[#8C8C8C]" onClick={onClick} disabled={disabled}>
```

Comment:
The back button looks great! Thanks for adding the hover color too, I appreciate the attention to detail. Quick code change though, instead of using `text-[#8C8C8C]`, you can use `text-medium-gray` from Tailwind config, and instead of `stroke-gray-500` use `stroke-dark-gray` (also in Tailwind config) so we can use consistent gray colors throughout the codebase! I'll figure out how to name these colors on Figma so that they'll show up in Dev Mode in the future

Reply:
Alright, will do